runhttpd.sh

```diff
@@ -2,11 +2,11 @@
 
 PROVISIONING_INTERFACE=${PROVISIONING_INTERFACE:-"provisioning"}
 HTTP_PORT=${HTTP_PORT:-"80"}
-HTTP_IP=$(ip -4 address show dev "$PROVISIONING_INTERFACE" | grep -oP '(?<=inet\s)\d+(\.\d+){3}' | head -n 1)
+HTTP_IP=$(ip -br addr show dev em2 | grep -Po "[^\s]+/[0-9]+" | grep -e "^fd" -e "\." | sed -e 's%/.*%%' | head -n 1)
```

Comment:
shouldn't we keep "$PROVISIONING_INTERFACE"  instead of em2 ?

Reply:
yup, thanks, that was a mistake from where I was testing, will fix